examples/siro_sandbox/README_VR.md

```diff
@@ -0,0 +1,218 @@
+| `habitat-lab` | This specific `habitat-lab` version must be installed. See [instructions](../../README.md#installation). The `main` branch is currently incompatible. |
+| Datasets | `python -m habitat_sim.utils.datasets_download --uids habitat_humanoids hab_spot_arm ycb hssd-hab --data-path data/` |
+| [hssd-models](https://huggingface.co/datasets/hssd/hssd-models) | Required for dataset processing. |
+| [habitat_humanoids](https://huggingface.co/datasets/ai-habitat/habitat_humanoids) | Use the `main` branch. |
+| [Server files](https://huggingface.co/datasets/ai-habitat/siro_fetch_extra_data) | Copy the files as-is to `data/`, following the directory structure. |
+| `websockets` | `pip install websockets` |
+
+#### Client
+
+| Requirements | Notes |
+|---|---|
+| Unity 2022.3.7f1 | Beware that a license may be required by your organization. |
+| Quest Headset | Tested on Quest Pro and Quest 3. Make sure that [developer mode](https://developer.oculus.com/documentation/native/android/mobile-device-setup/) is activated. |
+| [siro_hitl_unity_client](https://github.com/eundersander/siro_hitl_unity_client) | Use the `main` branch. See [instructions](https://github.com/eundersander/siro_hitl_unity_client/blob/main/README.md). |
+| Unity data folder | See [this section](#dataset-processing) for instructions for generating the data folder. |
+
+## Launch Commands
+
+The standard keyboard-mouse launch commands can be used with those differences:
+
+* The client-server application can be activated by including the `--remote-gui-mode` command-line argument.
+* Only `--app-state fetch` is supported.
+* Use the `examples/siro_sandbox/configs/fetch.yaml` configuration.
+
+#### Example
+
+```bash
+HABITAT_SIM_LOG=warning MAGNUM_LOG=warning \
+python examples/siro_sandbox/sandbox_app.py \
+--app-state fetch \
+--disable-inverse-kinematics \
+--never-end \
+--gui-controlled-agent-index 1 \
+--ang-speed "15" \
+--cfg examples/siro_sandbox/configs/fetch.yaml \
+--cfg-opts ~habitat.task.measurements.agent_blame_measure
+```
+
+## Unity Data Folder
+
+HSSD is not directly compatible with Unity. Furthermore, meshes must be simplified to run at an acceptable performance on the VR devices.
+
+Therefore, a script is provided so that you can process your HSSD scenes. Usage details are provided below.
+
+Note that the provided episodes support the following HSSD scenes:
+* 102344193
+* 102344280
+* 102817200
+* 103997424_171030444
+* 103997541_171030615
+
+### Dataset Processing
+
+#### Installation
+
+This step requires [Magnum](https://github.com/mosra/magnum) data processing tools. They are not built along with Habitat.
+
+It is recommended that you create a new `conda` environment so that it can be reused in the future without interfering with Habitat.
+
+1. Get the latest `magnum-tools` from [Magnum CI](https://github.com/mosra/magnum-ci/actions/workflows/magnum-tools.yml). Extract to a convenient location.
```

Comment:
I went to the link you provided but I just see a list of "workflow runs". It was not clear what to do or what to download. Also, we should specify platform requirements (don't these `magnum-tools` only work on linux?)